analyzers/tests/SonarAnalyzer.UnitTest/Rules/SingleStatementPerLineTest.cs

```diff
@@ -40,6 +42,21 @@ public class SingleStatementPerLineTest
                 .WithTopLevelStatements()
                 .Verify();
 
+        [TestMethod]
+        public void SingleStatementPerLine_Razor() =>
+            builderCS.AddSnippet(
+"""
+@if (true) { <p>Test</p> } <!-- Compliant FN -->
```

Comment:
The comment should be just FN. Since it cannot be compliant and a false negative at the same time.